frontend/src/components/search/SearchOpportunityStatus.tsx

```diff
@@ -68,7 +68,7 @@ const SearchOpportunityStatus: React.FC<SearchOpportunityStatusProps> = ({
 
   return (
     <>
-      <h4 className="margin-bottom-1">Opportunity status</h4>
+      <h2 className="margin-bottom-1">Opportunity status</h2>
```

Comment:
Does this make the heading larger? If so, you can apply utility classes so the h4 is h2-sized

Reply:
@andycochran - this did become slightly larger to h2 size - are you suggesting using a `div` with a utility class that makes it h2 sized? We can't have the `h4` tag anymore if we want to avoid the a11y flagged error

Reply:
Use the `h2` (not a `div`) with font size utility classes to make it look like an `h4`

Reply:
Hmm. Brainstorming here… 

We _could_ use the same `<h2 class="usa-accordion__heading">` so it matches the accordion headings. But that might be a hacky. 

`.usa-accordion__heading` uses: 
```
@include typeset($theme-accordion-font-family, $theme-body-font-size, 1);
```

…so we'd need to apply utility classes to make it match. 

I think just `<h2 class="font-sans-md">` will do it.

Reply:
`font-sans-xs` is what was closest to an `h4`